publisher-sdk/src/main/java/com/criteo/publisher/CriteoBannerView.java

```diff
@@ -25,14 +27,15 @@
 import androidx.annotation.VisibleForTesting;
 import com.criteo.publisher.integration.Integration;
 import com.criteo.publisher.integration.IntegrationRegistry;
+import com.criteo.publisher.model.AdSize;
 import com.criteo.publisher.model.BannerAdUnit;
 
 public class CriteoBannerView extends WebView {
 
   private static final String TAG = CriteoBannerView.class.getSimpleName();
 
   @Nullable
-  private final BannerAdUnit bannerAdUnit;
+  private BannerAdUnit bannerAdUnit;
```

Comment:
IMHO `final` is one of the best keyword in Java. It allows to have the strong guarantee that a value is constant.
You can keep this `final` by setting `bannerAdUnit = null` in the new constructor.